### ISSUE TITLE
[Feature request] Toggle fullscreen

### ISSUE THREAD

**Issue by shizonic** [xorg62/wmfs#100]
Is it possible to add a toggle fullscreen and/or toggle maximized function? I remembered to had such a function in wmfs version 1. Or I am wrong?

**Comment by xorg62** [comment]
I think there was a toggle_max yes, I'll look about this.

**Comment by shizonic** [comment]
Thanks! This would be very nice. :)

**Comment by shizonic** [comment]
Any news about this? :)

**Comment by xorg62** [comment]
yeah I tried some stuff locally, will finish it soon probably, sorry for the delay, I've been a bit busy ;>

**Comment by shizonic** [comment]
Hey you seriously don't have to excuse yourself. I really appreciate your work, your attention and quick responses! So thank you so much. :)

My question wasn't ment to hurry your just to get some status update. ;)

**Comment by shizonic** [comment]
Sorry for asking again but I really would like to have this feature. So still no progress there?

**Comment by xorg62** [comment]
Hello, it's alright I really am busy and hard to motivate these days, should have done this weeks ago...
So, you just want a way to toggle the selected window to maximized size, is that it? How are you expecting a new entering window to behave on a tag where the selected win is maximized? should it de-maximized the selected window or should it be integrated like it usually is in tiling, only unfocused and hidden? I imagine several ways to do it but I don't really know what your expectations are, I'm personnaly happy with using another tag or automatically tabbing windows together when i want them to remain maximized.

**Comment by shizonic** [comment]
Thanks again for all your effort.

I would like to have an option which lets me maximize (with visible bars) and toggle fullscreen (with hidden bars). So there should be two more functions.

If a new window appears on the same tag (which is either maximized or fullscreened) it should switch back to the normal tiled view and the new window should be focused. Is this possible?

By the way: Is possible to integrate an option to remember the last focused window on a tag too? So I can switch to another tag work there and switch back to the previous tag and the last focused window is focused again?

If you would integrate all of these features I would buy you a beer my friend. ;)

**Comment by shizonic** [comment]
Sorry for asking again (please say it if I should stop it) but what is the current state of this feature request? Thanks in advance! :)

**Comment by shizonic** [comment]
Still no progress here?
